src/k8s-engine/k3sHelper.ts

```diff
@@ -581,7 +581,11 @@ export default class K3sHelper extends events.EventEmitter {
       };
 
       console.log(`Updating kubeconfig ${ userPath }...`);
-      userConfig.loadFromFile(userPath);
+      try {
+        userConfig.loadFromFile(userPath);
+      } catch (_) {
+        // continue to merge into an empty userConfig == `{ contexts: [], clusters: [], users: [] }`
```

Comment:
Should we not at least log this? This could lead to data loss if we failed for reasons other than `ENOENT`.